include/fastdds/dds/domain/DomainParticipantFactory.hpp

```diff
@@ -88,23 +88,39 @@ class DomainParticipantFactory
      * If multiple DomainParticipant entities belonging to that domain_id exist,
      * then the operation will return one of them. It is not specified which one.
      * @param domain_id
-     * @return previously created DomainParticipant
+     * @return previously created DomainParticipant within the specified domain
      */
     RTPS_DllAPI DomainParticipant* lookup_participant(
             DomainId_t domain_id) const;
 
     /**
      * Returns all participants that belongs to the specified domain_id.
      * @param domain_id
-     * @return
+     * @return previously created DomainParticipants within the specified domain
      */
     RTPS_DllAPI std::vector<DomainParticipant*> lookup_participants(
             DomainId_t domain_id) const;
 
-    //! Fills qos with the default values.
+    /**
+     * @brief This operation retrieves the default value of the DomainParticipant QoS, that is, the QoS policies which will
+     * be used for newly created DomainParticipant entities in the case where the QoS policies are defaulted in the
+     * create_participant operation.
+     * The values retrieved get_default_participant_qos will match the set of values specified  on the last successful call
+     * to set_default_participant_qos, or else, if the call was never made, the default values.
+     * @param qos DomainParticipantQos to be set
+     * @return RETCODE_INCONSISTENT_POLICY if the Qos is not self consistent and RETCODE_OK if the qos is changed correctly.
```

Comment:
How could `RETCODE_INCONSISTENT_POLICY` be returned? Would that mean that the default qos is inconsistent?

Reply:
This method returns `RETCODE_INCONSISTENT_POLICY` if the user data vector size exceeds the max_user_data limit